### DISCUSSION TITLE
Hello from a recent participant in the MakeCode AP CSP PD

### DISCUSSION THREAD

**Discussion by RandyCHS** [microsoft/makecode-csp#70] [General]
Since it was looking pretty quite here, I thought I might get some discussion started with some past, current and future users of this curriculum.   I just finished attending the MakeCode AP CSP PD Workshop #2.  I also attended the same workshop last year, so this will be my 2nd year teaching with this curriculum.  So please feel free to ask me any questions about using this curriculum and/or your own experience with it.  I've hoping that as a community, we can work together to find best ways to apply and improve this curriculum as well as to better teach AP CSP in general.

**Comment by ketheredge** [comment]
Thanks Randy for starting the conversation. I am happy to join in to answer questions, discuss how to teach AP CSP better, and hear feedbac on the curriculum. :)
